checkstyle.xml

```diff
@@ -207,9 +207,7 @@
         <module name="OverloadMethodsDeclarationOrder"/>
         <module name="VariableDeclarationUsageDistance"/>
         <module name="CustomImportOrder">
```

Comment:
This custom import order isn't intuitive, not supported by default by IntelliJ, not used by azure-sdk-for-java, and 30 of the 48 checkstyle errors disappeared after we removed this.